html-aam/roles-generic.tentative.html

```diff
@@ -10,12 +10,44 @@
   <script src="/wai-aria/scripts/aria-utils.js"></script>
 </head>
 <body>
+<p>Tests contextual computedrole mappings for the <code>LI</code> element defined in <a href="https://w3c.github.io/html-aam/">HTML-AAM</a>, 
+   where the returned computed role is expected to change based on whether the <code>LI</code> element is an accessibility child of a list element, or not.<p>
```

Comment:
Have you diagnosed why you're getting these errors? It wasn't immediately clear, but debugging locally on the console is usually easiest and the same or similar error is consistent in all implementations.

```
ERROR message: Traceback (most recent call last):
  File "/Users/runner/work/1/s/tools/wptrunner/wptrunner/executors/executorwebdriver.py", line 807, in run_func
    self.result = True, self.func(self.protocol, self.url, self.timeout)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/work/1/s/tools/wptrunner/wptrunner/executors/executorwebdriver.py", line 997, in do_testharness
    raw_results = self.run_testdriver(protocol, url, timeout)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/work/1/s/tools/wptrunner/wptrunner/executors/executorwebdriver.py", line 929, in run_testdriver
    done, rv = handler(test_driver_message)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/work/1/s/tools/wptrunner/wptrunner/executors/base.py", line 755, in __call__
    return callback(url, payload)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/work/1/s/tools/wptrunner/wptrunner/executors/base.py", line 772, in process_action
    result = action_handler(payload)
             ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/work/1/s/tools/wptrunner/wptrunner/executors/actions.py", line 64, in __call__
    element = self.protocol.select.element_by_selector(selector)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/work/1/s/tools/wptrunner/wptrunner/executors/protocol.py", line 288, in element_by_selector
    raise ValueError(f"Selector '{element_selector}' matches multiple elements")
ValueError: Selector '#\74 ' matches multiple elements
```

Reply:
If you get stuck, let me know and I will download your branch to try it.

Reply:
It seems to be because the `<li>` elements on [lines 32 and 36](https://github.com/web-platform-tests/wpt/pull/49643/files#diff-e5c3523975723fd40fe7e2aa2949d19b95d66bd4e05f3f80e43aacc1f4d27a7eR32-R36) have identical `id` values.

I just changed one of them locally and the test ran successfully (with 1 failure for Chrome).

Reply:
It seems to be because the `<li>` elements on [lines 32 and 36](https://github.com/web-platform-tests/wpt/pull/49643/files#diff-e5c3523975723fd40fe7e2aa2949d19b95d66bd4e05f3f80e43aacc1f4d27a7eR32-R36) have identical `id` values.

I changed one of them locally and it ran successfully (with 1 failure in Chrome).

Reply:
thanks